src/data/chats/chat.rs

```diff
@@ -331,12 +326,11 @@ impl Chat {
             id: next_id + 1,
             role: Role::Assistant,
             username: Some(wanted_file.name.clone()),
-            entity: Some(ChatEntity::ModelFile(wanted_file.id.clone())),
+            entity: Some(ChatEntityId::ModelFile(wanted_file.id.clone())),
             content: "".to_string(),
         });
 
         self.state = ChatState::Receiving;
-        self.associated_entity = Some(ChatEntity::ModelFile(wanted_file.id.clone()));
```

Comment:
This line may pass unnoticed so I'm commenting it. Now, sending a message will not override the associated entity of the chat as `@` invoked models should not override the currently selected one on the selector.